src/LineSegment.cpp

```diff
@@ -2,6 +2,7 @@
 // Created by rolf on 15-5-19.
 //
 
+#include <HalfLine.h>
```

Comment:
These curly braces usually imply that the header is from an external library, which is not the case here (please be consistent).

Reply:
I will change that as well